week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,31 +1,39 @@
-// Add comments to explain what this function does. You're meant to use Google!
+// This function returns a number in the range of 0 (inclusive) and 1 (exclusive) and multiplies it by 10.
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10;
 }
 
-// Add comments to explain what this function does. You're meant to use Google!
+// This function joins the two parameters, w1 and w2.
 function s(w1, w2) {
-    return w1.concat(w2);
+  return w1.concat(w2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
-    // Write the body of this function to concatenate three words together
-    // Look at the test case below to understand what to expect in return 
+  return firstWord.concat(" ", secondWord, " ", thirdWord);
```

Comment:
This is correct, but I actually did not know this was possible!

Having done a little bit of research, I believe that using the `.concat()` method on **strings** shouldn't really be used any more because it can be quite slow.

Instead we should use the `+` operator:

```js
return firstWord + " " + secondWord + " " + thirdWord;
```